style.css

```diff
@@ -349,28 +379,27 @@ body {
 
 .form-container {
   position: absolute;
-  display: flex;
-  flex-direction: column;
-  width: 322px;
+  width: 320px;
   height: 813px;
   left: -2px;
-  top: 4129px;
+  top: 4205px;
+  background: #f9f9fa;
 }
```

Comment:
- [ ] Just this little change, your project is looking very good! 😄 
![Captura de Pantalla 2022-11-03 a la(s) 2 32 33 p  m](https://user-images.githubusercontent.com/86022690/199818035-106867fd-3e6c-4024-83d0-70f6d466482b.png)

Your footer is not positioned at the bottom of the container so it has a blank space under it (just in the mobile version). 

You can fix this by removing the height of your `.form-container` 👍